llarp/rpc/rpc_server.cpp

```diff
@@ -456,39 +456,54 @@ namespace llarp::rpc
                   {
                     auto mapExit = [=](service::Address addr) mutable {
                       ep->MapExitRange(range, addr);
+                      r->routePoker().Enable();
```

Comment:
this should be a result of the MapExitRange call, not the caller's responsibility.